ert_shared/storage/http_server.py

```diff
@@ -147,6 +153,24 @@ def data(self, data_id):
             else:
                 return str(data)
 
+    def datas(self):
+        ids = request.args.get("ids").split(",")
+
+        def generator():
```

Comment:
sould probably yield a 404 if the id is not present in the database?

Reply:
I'll see what I can do

Reply:
I have added the code to give a 404 to the consumers of this method if one asks for the blobs for a non existent parameter or response. Will make this method private.